tableconvert/src/tableconvert.h

```diff
@@ -27,7 +27,7 @@
 
 #include <geanyplugin.h>
 #include <gtk/gtk.h>
-
+#include <geany/filetypes.h>
```

Comment:
This is not required. I removed it and can still build Geany-Plugins. Also it makes the Travis-CI build fail for some reason (but I don't know why).